libs/shared/utils/src/lib/formatNationalId.ts

```diff
@@ -0,0 +1,7 @@
+export const formatNationalId = (nationalId: string): string => {
```

Comment:
The kennitala package already does this, so this isn't needed:

```typescript
import * as kennitala from 'kennitala'

const formattedNationalId = kennitala.format(someNationalId)
```

Reply:
Ah of course. Copied this utility from another place in the monorepo. Should probably get rid of those too, just didn't want to pull in more code reviewers in this PR.